README.md

```diff
@@ -1,25 +1,59 @@
-# Microverse HTML CSS Javascript-Capstone
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Kindly include in the video pieces of code you are interested in as in the project requirement.
![image](https://user-images.githubusercontent.com/24703171/168912138-2944da83-e03b-45df-b0ae-18c307db4658.png)

Reply:
thanks for all helpful advice, I will follow them